iothub/device/src/IotHubModuleClient.cs

```diff
@@ -157,17 +159,17 @@ public async Task SendEventAsync(string outputName, Message message, Cancellatio
         /// <exception cref="OperationCanceledException">Thrown when the operation has been canceled.</exception>
         public async Task SendEventBatchAsync(string outputName, IEnumerable<Message> messages, CancellationToken cancellationToken = default)
         {
-            try
-            {
-                if (Logging.IsEnabled)
-                    Logging.Enter(this, outputName, messages, nameof(SendEventBatchAsync));
+            if (Logging.IsEnabled)
+                Logging.Enter(this, outputName, messages, nameof(SendEventBatchAsync));
 
-                ValidateModuleTransportHandler("SendEventBatchAsync for a named output");
+            Argument.AssertNotNullOrWhiteSpace(outputName, nameof(outputName));
 
-                Argument.AssertNotNullOrWhiteSpace(outputName, nameof(outputName));
+            var messagesList = messages?.ToList();
+            Argument.AssertNotNullOrEmpty(messagesList, nameof(messages));
 
-                var messagesList = messages?.ToList();
-                Argument.AssertNotNullOrEmpty(messagesList, nameof(messages));
+            try
+            {
+                ValidateModuleTransportHandler("SendEventBatchAsync for a named output");
```

Comment:
see line 172 and 130, should both of them be consistently in or out of the try block?

Reply:
Yep, this should be after line 165.

Reply:
Moved both out of the try block